mne_bids/write.py

```diff
@@ -669,6 +672,8 @@ def _sidecar_json(raw, task, manufacturer, fname, datatype, overwrite=False,
         ('DigitizedHeadPoints', digitized_head_points),
         ('MEGChannelCount', n_megchan),
         ('MEGREFChannelCount', n_megrefchan)]
+    if emptyroom_fname is not None:
+        ch_info_json_meg.append(('AssociatedEmptyRoom', str(emptyroom_fname)))
```

Comment:
I think this might be a bug if `emptyroom_fname` is a BIDSPath -> should probably be `str(emptyroom_fname.fpath)` then?

Reply:
Actually no, `str(BIDSPath)` does the right thing.